tests/functions/user_delete_test.php

```diff
@@ -59,6 +59,7 @@ protected function setUp()
 		$passwords_helper = new \phpbb\passwords\helper;
 		// Set up passwords manager
 		$passwords_manager = new \phpbb\passwords\manager($config, $passwords_drivers, $passwords_helper, array_keys($passwords_drivers));
+		$dispatcher = new phpbb_mock_event_dispatcher();
```

Comment:
The dispatcher is already defined as `$phpbb_dispatcher` a few lines above this. As mentioned in my comment, you don't actually need to pass the dispatcher to the class via the constructor though.